### ISSUE TITLE
Make dark mode toggleable

### ISSUE THREAD

**Issue by marleysudbury** [marleysudbury/marleysudbury.github.io#15]
Currently dark or light mode is determined by the system settings, but for improved UX / accessibility it should be possible for the user to override this.

**Comment by marleysudbury** [comment]
[https://stackoverflow.com/questions/56300132/how-to-override-css-prefers-color-scheme-setting](https://stackoverflow.com/questions/56300132/how-to-override-css-prefers-color-scheme-setting)

This looks like a helpful resource.
